vizro-ai/docs/pages/user-guides/install.md

```diff
@@ -125,12 +123,10 @@ You should see a return output of the form `x.y.z`.
 
 ## Upgrade
 
+Check the [release notes](https://github.com/mckinsey/vizro/blob/main/vizro-ai/CHANGELOG.md) for any notable breaking changes before migrating an existing project.
+
 To change the version of Vizro-AI installed:
 
 ```bash
 pip install -U vizro_ai
 ```
```

Comment:
Wondering if we need this entire section? Is this something people commonly include in the docs?

Reply:
I think it's worth including -- I always have to look it up (though maybe I'm the only one!)